### PR TITLE
Regenerate service files

### PR DESCRIPTION
**Pull request by JeffreyDevloo**

### CODE REVIEW

File: ovs/extensions/services/systemd.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ class Systemd(object):
     Contains all logic related to Systemd services
     """
     _logger = LogHandler.get('extensions', name='service-manager')
+    SERVICE_CONFIG_KEY = '/ovs/framework/hosts/{0}/services/{1}'
```
Comment (kvanhijf) [inline ovs/extensions/services/systemd.py:34]: Please add same functionality to the ASD-manager repository

File: ovs/extensions/services/systemd.py [hunk 2]
```diff
@@ -105,6 +106,41 @@ def add_service(name, client, params=None, target_name=None, startup_dependency=
         return params
 
     @staticmethod
+    def regenerate_service(name, client, target_name):
+        """
+        Regenerates the service files of a service.
+        :param name: Template name of the service to regenerate
+        :type name: str
+        :param client: Client on which to add the service
```
Comment (kvanhijf) [inline ovs/extensions/services/systemd.py]: We don't add a service (2)

File: ovs/extensions/services/systemd.py [hunk 3]
```diff
@@ -105,6 +106,41 @@ def add_service(name, client, params=None, target_name=None, startup_dependency=
...
+        :type client: ovs.extensions.generic.sshclient.SSHClient
+        :param target_name: The current service name eg ovs-volumedriver_flash01.service
+        :type target_name: str
+        :return: None
+        :rtype NoneType
+        """
+        configuration_key = Systemd.SERVICE_CONFIG_KEY.format(System.get_my_machine_id(client),
+                                                              target_name.replace('ovs-', ''))
```
Comment (kvanhijf) [inline ovs/extensions/services/systemd.py]: Please use Toolbox.remove_prefix

File: ovs/extensions/services/upstart.py [hunk 4]
```diff
@@ -98,6 +99,41 @@ def add_service(name, client, params=None, target_name=None, startup_dependency=
         return params
 
     @staticmethod
+    def regenerate_service(name, client, target_name):
+        """
+        Regenerates the service files of a service.
+        :param name: Template name of the service to regenerate
+        :type name: str
+        :param client: Client on which to add the service
```
Comment (kvanhijf) [inline ovs/extensions/services/upstart.py]: Same shizzle

File: ovs/extensions/services/upstart.py [hunk 5]
```diff
@@ -98,6 +99,41 @@ def add_service(name, client, params=None, target_name=None, startup_dependency=
...
+        :type client: ovs.extensions.generic.sshclient.SSHClient
+        :param target_name: The current service name eg ovs-volumedriver_flash01.service
+        :type target_name: str
+        :return: None
+        :rtype NoneType
+        """
+        configuration_key = Upstart.SERVICE_CONFIG_KEY.format(System.get_my_machine_id(client),
+                                                              target_name.replace('ovs-', ''))
```
Comment (kvanhijf) [inline ovs/extensions/services/upstart.py]: indentation? and Toolbox.remove_prefix please

**Review by kvanhijf — CHANGES_REQUESTED** [review]

**Comment by JeffreyDevloo** [comment]
https://github.com/openvstorage/alba-asdmanager/pull/207

File: ovs/extensions/services/systemd.py [hunk 6]
```diff
@@ -105,6 +106,40 @@ def add_service(name, client, params=None, target_name=None, startup_dependency=
...
+        """
+        Regenerates the service files of a service.
+        :param name: Template name of the service to regenerate
+        :type name: str
+        :param client: Client on which to regenerate the service
+        :type client: ovs.extensions.generic.sshclient.SSHClient
+        :param target_name: The current service name eg ovs-volumedriver_flash01.service
+        :type target_name: str
+        :return: None
+        :rtype NoneType
```
Comment (kvanhijf) [inline ovs/extensions/services/systemd.py]: colon

File: ovs/extensions/services/upstart.py [hunk 7]
```diff
@@ -98,6 +99,40 @@ def add_service(name, client, params=None, target_name=None, startup_dependency=
...
+        """
+        Regenerates the service files of a service.
+        :param name: Template name of the service to regenerate
+        :type name: str
+        :param client: Client on which to regenerate the service
+        :type client: ovs.extensions.generic.sshclient.SSHClient
+        :param target_name: The current service name eg ovs-volumedriver_flash01.service
+        :type target_name: str
+        :return: None
+        :rtype NoneType
```
Comment (kvanhijf) [inline ovs/extensions/services/upstart.py]: colon

**Review by kvanhijf — CHANGES_REQUESTED** [review]

**Review by kvanhijf — APPROVED** [review]
Jerry - 7 times - Devloo succeeded

**Comment by kvanhijf** [comment]
https://github.com/openvstorage/framework/pull/1348 --> openvstorage-2.7.10-dev.1484665298.dc8c2ea
https://github.com/openvstorage/alba-asdmanager/pull/207 --> openvstorage-sdm-1.6.10-dev.1484665413.4f101ba

**Comment by khenderick** [comment]
openvstorage-2.7.10-1

**Pull request merged**